### ISSUE TITLE
Test content appears to not match test name

### ISSUE THREAD

**Issue by davidillsley** [dwp/nino-format-validation#2]
Hi, I'm trying to understand the test [validateReturnDayOfWeekGivesThursdayWhenNinoIsValidAndHasLessThanTwentyAsLastNumbers](https://github.com/dwpdigitaltech/nino-format-validation/blob/master/src/test/java/gov/dwp/utilities/formatvalidation/NinoValidatorTest.java#L113). The name of the test suggests it's checking some behaviour when the last numbers in the NINO is less than 20, but the NINO body used is "AA370773".
Is there some special logic around Thursdays (and or '20'), or is the rule as described on [wikipedia](https://en.wikipedia.org/wiki/National_Insurance_number)?

**Comment by HealthPDU** [comment]
so sorry this comment has remained unanswered @davidillsley; we never received an alert which is unfortunate.  You're spot on, the rule works as described on the Wikipedia page.

I've checked our commit history...  When the test was introduced we were using a correctly formatted random `NINO`.  Before publishing, our compliance checks highlighted that the nino _could_ inadvertently belong to a citizen so we replaced all instances with a internal test `NINO`.

The test was updated, the function name only partially.  This is exactly the type of scrutiny we were hoping for, many thanks 👍
